test/refactoring/ConvertAsync/convert_bool.swift

```diff
@@ -0,0 +1,271 @@
+// REQUIRES: objc_interop
+// REQUIRES: concurrency
+
+// RUN: %empty-directory(%t)
+
+import Foundation
+import ConvertBoolObjC
+
+func boolWithErr(completion: (Bool, Error?) -> Void) {}
+func multipleBoolWithErr(completion: (String?, Bool, Bool, Error?) -> Void) {}
+func optionalBoolWithErr(completion: (String?, Bool?, Bool, Error?) -> Void) {}
```

Comment:
Isn't actually a test for this one

Reply:
Ah, good spot!

Reply:
Turns out there was a bit of logic missing needed to handle this case! Updated now.